ci-operator/config/red-hat-data-services/caikit-tgis-serving/red-hat-data-services-caikit-tgis-serving-rhods-1.32.yaml

```diff
@@ -0,0 +1,39 @@
+base_images:
+  base:
+    name: text-generation-inference
+    namespace: red-hat-data-services
+    tag: rhods-1.32
```

Comment:
This should be matched with this :https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/text-generation-inference/red-hat-data-services-text-generation-inference-rhods-1.32.yaml#L17

You can update this line or text-generation-inferenece repo.